.credo.exs

```diff
@@ -21,10 +21,23 @@
         # You can give explicit globs or simply directories.
         # In the latter case `**/*.{ex,exs}` will be used.
         #
-        included: ["lib/", "src/", "test/", "web/", "apps/"],
-        excluded: [~r"/_build/", ~r"/deps/"]
+        included: [
+          "lib/",
+          "src/",
```

Comment:
sorry about the credo re-gen, but didn't have the WithClause check and I wanted to disable it.
here are the credo customizations after gen
https://github.com/NFIBrokerage/harness/pull/15/commits/154d964d85ccdf128388d020c919d35badfbd478